pkgs/by-name/od/odmpy/package.nix

```
@@ -0,0 +1,84 @@
+{
+  bash,
+  ffmpeg,
+  lib,
+  python3,
+  fetchFromGitHub,
+  writableTmpDirAsHomeHook,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```
a bit easier to override
(same suggestion for all instances of python3.pkgs)